forms/error-helpers.js

```diff
@@ -21,6 +21,48 @@ define(function (require) {
     };
   };
 
+  function makeCustomError(code, val, errorString){
+    var ret = {};
+
+    if ( errorString === undefined){
+      errorString = blinkFormsError.toErrorString(val) || val;
+    }
+
+    ret.code = code;
+    ret[code] = val;
```

Comment:
I added 'CUSTOM' to that list, and made this function mostly for convenience when making custom errors. I figured the rest are pretty much internal errors ??